client/job.lua

```diff
@@ -300,7 +300,7 @@ CreateThread(function()
     end
 
     for _, coords in pairs(sharedConfig.locations.helicopter) do
-        createGarage(config.authorizedHelicopters, Lang:t('info.heli_plate'), coords)
+        createGarage(config.authorizedHelicopter, Lang:t('info.heli_plate'), coords)
```

Comment:
authorizedHelicopters actually sounds like it should be the correct name? Maybe change the config to match instead of changing here.